.github/workflows/ci.yml

```diff
@@ -18,10 +18,7 @@ jobs:
 
       - run: npm ci
 
-      - name: Check code format
-        uses: creyD/prettier_action@v4.2
-        with:
-          prettier_options: --check client/src snippets syntaxes
+      - run: npm run format:check
```

Comment:
Prior to this change, the action would `cd` into another directory and mess up subsequent steps.